docs/Component.md

```diff
@@ -70,7 +70,7 @@ when permission is denied. If you want to check authorization and get a boolean
 result you can use the `can()` method:
 
 ```php
-if ($this->Authorization->can($article, 'update')) {
+if ($this->Authorization->can($article, 'update')->getStatus()) {
```

Comment:
In the paragraph above it mentions that if you want a boolean you can use `can()`. Is that no longer the case?